detox/src/devices/drivers/AndroidDriver.test.js

```diff
@@ -91,8 +91,8 @@ describe('Android driver', () => {
         await promise;
         fail('Expected an error and none was thrown');
       } catch (e) {
-        expect(e.message).toContain('DetoxRuntimeError: Failed to run application on the device');
-        expect(e.message).toContain(`Native stacktrace dump: ${mockInstrumentationLogsParserClass.INSTRUMENTATION_STACKTRACE_MOCK}`);
+        expect(e.toString()).toContain('DetoxRuntimeError: Failed to run application on the device');
```

Comment:
It was incorrect from my side to put `DetoxRuntimeError:` inside Error.prototype.message.
I started seeing at some places `DetoxRuntimeError: DetoxRuntimeError: ...`, so I decided to fix it.